src/phases/query/QueryStatsVariants.tmpl

```diff
@@ -0,0 +1,157 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: >
+  This is a template of predefined variants for QueryStats workloads.
```

Comment:
What exactly is a "variant" - can you elaborate here? It seems like this is a new layer of abstraction, as opposed to the other files here which are common to genny things. So... needs very clear documentation - it also shares a name with the 'evergreen variant' terminology - is there perhaps a better name? Did you consider others? It seems maybe like a "MacroPhase"? or something?